fedimint-core/src/time.rs

```diff
@@ -27,21 +27,22 @@ mod wasm {
         Encodable,
         Decodable,
     )]
-    pub struct SystemTime(Duration);
+    pub struct SystemTime(std::time::SystemTime);
```

Comment:
Do we actually still need the newtype? Seems like we could just use std `SystemTime` directly? Maybe except for `now` which we could amend using a custom trait.

Reply:
IMO switching from std SystemTime to fedimint SystemTime should not require any other changes. If we add a trait that also needs to be imported :shrug: